test/shared/policies/deserializationPolicyTests.ts

```diff
@@ -62,7 +62,7 @@ describe("deserializationPolicy", () => {
     assert.deepStrictEqual(response.parsedBody, [123, 456, 789]);
   });
 
-  it("should parse a JSON response body with an uppercase Content-Type", async function() {
+  it("should parse a JSON response body with an uppercase Content-Type", async () => {
```

Comment:
arrow functions do not work well in mocha https://glebbahmutov.com/blog/test-closures/, https://github.com/mochajs/mocha/issues/2018

Reply:
Changed it back.